tests/spec/w3c/linter-rules/required-sections-spec.js

```diff
@@ -0,0 +1,114 @@
+"use strict";
+
+import {
+  errorFilters,
+  flushIframes,
+  makeRSDoc,
+  makeStandardOps,
+} from "../../SpecHelper.js";
+
+describe("w3c — required-sections", () => {
+  afterAll(() => {
+    flushIframes();
+  });
+
+  const linterErrors = errorFilters.filter(
+    "w3c/linter-rules/required-sections"
+  );
+
+  it("doesn't generate errors for non-rec-track that don't have the required sections", async () => {
+    const ops = makeStandardOps({
+      lint: { "required-sections": true },
+      specStatus: "ED",
+    });
+    const doc = await makeRSDoc(ops);
+    const errors = linterErrors(doc);
+    expect(errors).toHaveSize(0);
+  });
+
+  it("does nothing if disabled", async () => {
+    const ops = makeStandardOps({ lint: { "required-sections": true } });
```

Comment:
Testing if disabled:
```suggestion
    const ops = makeStandardOps({ lint: { "required-sections": false } });
```

Reply:
Also let's test it on a recTrack document?